bootstrap/create-tf-state-bucket.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
I've set executable on this file via `git update-index --chmod=+x ./bootstrap/create-tf-state-bucket.sh` - I'll include in next commit 👍

Reply:
👍 Great thanks - actually I can use that same approach to avoid using a `chmod` in the `travis.yml` to run the deploy script